src/java/org/apache/cassandra/config/Config.java

```diff
@@ -1283,4 +1283,21 @@ public static void log(Config config)
     public double severity_during_decommission = 0;
 
     public StorageCompatibilityMode storage_compatibility_mode;
+
+    public enum CQLStartTime
+    {
+        REQUEST, // uses a timestamp that represent the start of processing of the request
+        QUEUE    // uses a timestamp that represents when the request was enqueued
+    }
+    public volatile CQLStartTime cql_start_time = CQLStartTime.REQUEST;
```

Comment:
nit: For this one, I guess we should clarify that it only applies to the replica timeout and `native_transport_timeout` still governs the overall coordinator-level request.